docs/en/reference/information_schema/task_runs.md

```diff
@@ -0,0 +1,24 @@
+---
+displayed_sidebar: "English"
+---
+
+# task_runs
+
+`task_runs` provides information about the execution of asynchronous tasks.
+
+The following fields are provided in `task_runs`:
+
+| **Field**     | **Description**                                              |
+| ------------- | ------------------------------------------------------------ |
+| QUERY_ID      | ID of the query.                                             |
+| TASK_NAME     | Name of the task.                                            |
+| CREATE_TIME   | Time when the task is created.                               |
+| FINISH_TIME   | Time when the task finished.                                 |
+| STATE         | State of the task, including PENDING, RUNNING, FAILED, and SUCCESS. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/979710ac-4273-4d3b-92d0-a3440865fa43)
-> Valid values: `PENDING`, `RUNNING`, `FAILED`, and `SUCCESS`.